main/modes/mode_gamepad.c

```diff
@@ -55,15 +70,39 @@ void gamepadTouchCb(touch_event_t* evt);
 void gamepadAccelCb(accel_t* accel);
 void gamepadReportStateToHost(void);
 
+void setGamepadMainMenu(void);
+void gamepadMainMenuCb(const char* opt);
+void gamepadMenuLoop(int64_t elapsedUs);
+void gamepadMenuButtonCb(buttonEvt_t* evt);
+void gamepadMenuTouchCb(touch_event_t* evt);
+void gamepadMenuAccelCb(accel_t* accel);
+void gamepadStart(display_t* disp, gamepadType_t type);
+
 //==============================================================================
 // Variables
 //==============================================================================
 
+
+static const char str_gamepadTitle[] = "Gamepad Type";
+static const char str_pc[] = "PC";
```

Comment:
`"PC"` and `"NS"` too sparse to be clearly understood. `"PC Gamepad"` and `"N. Switch"` would be clearer, assuming there is space. I'm open to other ideas too.

Reply:
Just trying to avoid using any trademarks.

Reply:
I get that, and that's why I didn't want "Nintendo Switch" (space aside). No one's going to understand "NS" though. I didn't, except that I knew exactly what it was already.

Reply:
Changed to just "Switch"